internal/pkg/helpers/app.go

```diff
@@ -11,6 +13,25 @@ import (
 	"github.com/joshsoftware/profile_builder_backend_go/internal/pkg/errors"
 )
 
+func SendRequest(ctx context.Context, url, access_token string) ([]byte, error) {
+	serverRequest, err := http.NewRequestWithContext(ctx, "GET", url, nil)
+	if err != nil {
+		return nil, errors.ErrInvalidRequestData
```

Comment:
here we miss the actual error, log the error

Reply:
catched the actual error and log the error